Estrutura inicial "Seja um colaborador"

EduLeal opened the pull request:
Coloquei mais detalhes na descrição do commit

EduLeal committed:
Merge pull request #2 from Jovens-do-futuro/main

Merge pull request #1 from Jovens-do-futuro/edu-dev

EduLeal committed:
Merge pull request #4 from Jovens-do-futuro/main

Merge da branch principal pra minha área de desenvolvimento

EduLeal committed:
feat: estrutura básica

Não existia a pasta do "Seja um colaborador" até então, também foi criada uma página em HTML com estrutura básicas e estilização simples em CSS, também criei uma pasta pras imagens, uma é o ícone da página e a outra é uma imagem temporária que futuramente pode ser substituída por imagens dos serviços sendo realizados

luizbrito6 commented on pages/seja-colaborador/style.css:
```diff
@@ -0,0 +1,25 @@
+.header {
+  padding: 15px 30px;
+  font-size: 18px;
```
Depois no figma acho legal a gente criar classes globais de estilos de texto, aí fica bem padronizado. 

A ideia seria fazer algo assim, ai cada classe referenciaria um estilo de texto: 
![image](https://github.com/Jovens-do-futuro/website/assets/112624030/86d63b0d-05c6-4834-8d17-d25bdf2c5163)


Segue link do figma: https://www.figma.com/file/WqjF7xZzzv3rp8Mmw2E2sv/Style-Guidelines-(Community)?type=design&node-id=25-32&mode=design&t=POw8wukItQrehuwJ-0

luizbrito6 commented on pages/seja-colaborador/index.html:
Edu depois confirma qual o objetivo dessa página, ela vai ser referente ao contexto de doações? Depois confirma no map do site no figjam que a gente montou junto no fim de semana

luizbrito6 commented on pages/seja-colaborador/index.html:
```diff
@@ -0,0 +1,37 @@
+    <link rel="shortcut icon" type="imagex/png" href="./img/icone.ico" />
+  </head>
+  <body>
+    <div class="header">
```
Edu eu joguei os conceitos de header, main e footer sem explicar corretamente. Segue um vídeo e uma doc acho que fica melhor pra entender: 

- https://www.youtube.com/watch?v=IHMOu_KEW-0&pp=ygUPaHRtbCBzZW1hbnRpY28g
- https://www.devmedia.com.br/html-semantico-conheca-os-elementos-semanticos-da-html5/38065


Ai no caso seria importante deixar o header e footer vazios por enquanto pq vamos criar eles via js. As informaçoes da página vão dentro da main. Seria bom a gente alinhar essa estrutura com o grupo todo.

luizbrito6 replied:
@PedroFnseca

EduLeal replied:
Beleza, vou dar uma olhada nos vídeos e na documentação e arrumo assim que possível, muito obrigado pelo retorno :)

EduLeal committed:
fix: design básico figma

Tirei as coisas do layout anterior, limpei a header e coloquei as ids e classes que estão no figma

PedroFnseca commented:
@luizbrito6 aprova ae
